template/GOVERNANCE.md.tpl

```diff
@@ -0,0 +1,33 @@
+# Project governance
+
+This document lays out the guidelines under which the AWS Controllers for Kubernetes (ACK) project will be governed.
+The goal is to make sure that the roles and responsibilities are well defined and clarify on how decisions are made.
+
+## Roles
+
+In the context of ACK, we consider the following roles:
+
+* __Users__ ... everyone using ACK, typically willing to provide feedback on ACK by proposing features and/or filing issues.
+* __Contributors__ ... everyone contributing code, documentation, examples, testing infra, and participating in feature proposals as well as design discussions. Code contributions will require a Developer Certificate of Origin (DCO).
+*	__Maintainers__ ... are responsible for engaging with and assisting contributors to iterate on the contributions until it reaches acceptable quality. Maintainers can decide whether the contributions can be accepted into the project or rejected. Any active contributor meeting the project quality can be made a Maintainer by the Advisory Board.
+*	__Advisory Board__ ... is responsible for defining the guidelines and processes that the project operates under.
```

Comment:
Probably this entire section can be removed. Possibly we want to rewrite it to remain relevant but otherwise I don't think anyone will refer to this.